b2b_personalization/infra/prod/eu-central-1/b2b/sagemaker_multilingual_e5_large/terragrunt.hcl

```
@@ -0,0 +1,21 @@
+include "root" {
+  path = find_in_parent_folders()
+}
+
+include "env" {
+  path = "${get_terragrunt_dir()}/../../../../_env/sagemaker_endpoint.hcl"
+}
+
+prevent_destroy = true
+
+inputs = {
+  create_endpoint = true
+  endpoint_config_production_variants = [
+    {
+      initial_instance_count = 1
+      # instance_type          = "ml.c5.xlarge"
+      # remember to adjust the number of workers in the model_config.json
+      instance_type = "ml.t2.large"
+      variant_name  = "cpu"
```

```suggestion
      variant_name  = "t2-large"
```

I'm now leaning towards using the instance type. it is becoming handy if you want to compare the metrics for different instance types